UnityProject/Assets/StreamingAssets/config/gameConfig.json

```diff
@@ -35,5 +35,7 @@
 	
 	"LowPopLimit" : 20,
 
-	"LowPopCheckTimeAfterRoundStart" : 300
+	"LowPopCheckTimeAfterRoundStart" : 300,
+
+	"RebootOnAverageFPSOrLower" : 35
```

Comment:
make it MUCH lower. Server fps aren't the same as the fps you see while playing, I think even 10 is good server fps when  a lot of stuff is happening.

Reply:
20 seems like a better value since thats where the server dips a lot most of the time especially after a profile.